packages/util/src/units.ts

```diff
@@ -18,3 +18,21 @@ export function formatBigDecimal(
   const zerosPostDecimal = String(maxDecimalFactor).length - 1 - String(fraction).length
   return `${full}.${'0'.repeat(zerosPostDecimal)}${fraction}`
 }
+
+export class Units {
```

Comment:
Could you add docs to be explicit that the input amount is the amount of the unit, and that the returned value is in `wei`?

Also, I think a guard should be added to prevent negative numbers as input